src/templates/Vue InstantSearch with Vite/.template.js

```diff
@@ -0,0 +1,18 @@
+const install = require('../../tasks/node/install');
+const teardown = require('../../tasks/node/teardown');
+
+module.exports = {
```

Comment:
maybe instead of "with vite", it should lead with "with vue 3", what do you think?

Reply:
It's okay with me, I was not sure how opinionated we could be by chosing Vite over other tools.

Reply:
We're opinionated in the tools for the other examples too, so I don't think this really has an impact. As long as it works on codesandbox easily, any tool is fine